__tests__/api.test.js

```diff
@@ -262,6 +262,23 @@ describe("/api/articles/:article_id/comments", () => {
   });
 });
 
+describe("/api/users", () => {
+  test("GET:200 return all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const users = body.users;
+        expect(users.length).toBe(4);
+        users.forEach((user) => {
+          expect(typeof user.username).toBe("string");
```

Comment:
I'd rather you used .objectContaining or .toMatchObject instead of looking at the types of all these values. Have a go refactoring or using them in future tickets